app/utils/sort.ts

```diff
@@ -0,0 +1,11 @@
+import { sortBy } from 'lodash-es';
+
+interface CustomFormField {
+    fieldIdentifier: string,
+    isComplex: boolean
+}
+
+export function sortCustomFormFields(fields: CustomFormField[], fieldOrder: string[]): CustomFormField[] {
+    fieldOrder = [...fieldOrder].reverse();
+    return sortBy(fields, 'isComplex', item => -fieldOrder.indexOf(item.fieldIdentifier));
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5875345)